vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
 )
 ```
 
-## Coverage Diagnostics
-
-### Match Success Rate
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-## PACTA Target Market Share Metrics
-
-### Sector-level technology mix
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-### Technology-level volume trajectory
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
-
-## PACTA SDA Metric
-
-### Sector-level emission intensity
-
-* Name (of metric)
-* Formulae
-* Description
-* Possible action(s)/ impact
-* Use in combination with other metrics
-* Time horizon
-* Geographic relevance
-* Sectoral relevance
-* Plot demo (when applicable)
+## Target Market Share
+
+[brief description of Target Market Share metric]
```

Comment:
This should be completed.
@cjyetman presumably this, and all other content in [ ] brackets are things that you expect @jacobvjk to fill out?